packages/d/diffoscope/package.yml

```diff
@@ -0,0 +1,41 @@
+name       : diffoscope
+version    : 264
+release    : 1
+source     :
+    - https://diffoscope.org/archive/diffoscope-264.tar.bz2 : bdffe8adabbcb45ae57e54dad9b1821230022fd7355a2e5233a0d44cdf93eb09
+homepage   : https://diffoscope.org/
+license    : GPL-3.0-or-later
+component  : programming.python
+summary    : In-depth comparison of files, archives, and directories
+description: |
+    diffoscope tries to get to the bottom of what makes files or directories different. It will recursively unpack archives of many kinds and transform various binary formats into more human-readable form to compare them. It can compare two tarballs, ISO images, or PDF just as easily.
+builddeps  :
+    - python-build
+    - python-installer
+    - python-packaging
+    - python-wheel
+rundeps    :
+    - binutils
+    - bzip2
+    - cpio
+    - diffutils
+    - dtc
+    - gettext
+    - gzip
+    - libxmlb
+    - llvm
+    - lz4
+    - p7zip
+    - python-argcomplete
+    - python-defusedxml
+    - python-libarchive-c
+    - python-magic
+    - tar
+    - unzip
+    - xz
+    - zstd
+build      : |
+    %python3_setup
+install    : |
```

Comment:
There's a man page as well that you could add. `doc/diffoscope.1`

Reply:
it is kind of broken. it needs to be built first (`make -C doc`). it need `help2man` and `python-docutils`. I added those in builddeps, but it keeps failing said `rst2man` command not found and `help2man` bla..bla... So i removed those.

Reply:
I fixed it. It needed `help2man` and `python-docutils` like you thought, but it also tried to run `bin/diffoscope` from the build root and that failed with a ` No module named 'libarchive' No module named 'libarchive'` error. That was fixed by adding `python-libarchive-c` as a builddep as well.